trunner/runners/common.py

```diff
@@ -244,11 +254,14 @@ def from_pexpect(cls, pexpect_fd):
         obj.plo = pexpect_fd
         return obj
 
-    def interrupt_counting(self):
-        """ Interrupts timer counting to enter plo """
-        self.plo.expect_exact('Waiting for input', timeout=3)
+    def send_empty(self):
         self.plo.send('\r\n')
-        self.plo.expect_exact('\n')
+
+    def interrupt_counting(self, catch_statement):
```

Comment:
IMHO this should be named `enter_bootloader`

Reply:
Finally not needed argument.